traefik/tests/service-config_test.yaml

```diff
@@ -13,6 +13,13 @@ tests:
     asserts:
       - hasDocuments:
           count: 0
+  - it: should fail when trying to use this Chart with Traefik Proxy v2
```

Comment:
OK, now I see, but it should not be specifically checked in services.

Reply:
Fixed with 10f9102